libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -290,6 +290,15 @@ const AP_Param::GroupInfo AP_GPS::var_info[] = {
     AP_GROUPINFO("BLEND_TC", 21, AP_GPS, _blend_tc, 10.0f),
 #endif
 
+#if GPS_UBLOX_MOVING_BASELINE
+    // @Param: UBX_OPTIONS
+    // @DisplayName: Ublox options
+    // @Description: Additional options for UBlox backend
+    // @Bitmask: 0:Use UART2 for moving baseline
+    // @User: Advanced
+    AP_GROUPINFO("UBX_OPTIONS", 22, AP_GPS, _ublox_options, 0),
```

Comment:
Maybe we should call this driver ( `DRV_OPTIONS` ) instead? This way other backends can actually grab bits/takeover the meaning here if they want? I'm not really thrilled with the invasive upwards propagation of ublox features escaping past the driver. (It also still seems like this is not really needed and probably could be covered with an additional pair of `TYPE` entries, but I do grant that eventually that gets excessive)

Reply:
done